src/app/project/comments/comments.component.scss

```diff
@@ -22,7 +22,8 @@
   background-color: #001330;
 
   .hero-banner__container {
```

Comment:
Possibly replace literal hex values with variables if there's time (line 22 for example)